lib/assets/schemas/umm-var-form.json

```diff
@@ -98,7 +98,7 @@
       ]
     },
     {
-      "id": "fill_value",
+      "id": "fill_values",
```

Comment:
Yep, it did. I noticed the title on the edit form didn't match once I added the form titles. Sets and Services as well